src/sections/Welcome.js

```diff
@@ -0,0 +1,109 @@
+import { css } from "@emotion/core"
+import styled from "@emotion/styled"
+import React from "react"
+import Particles from "react-tsparticles"
+import Button from "../components/Button"
+
+function Welcome(props) {
+  const ParticlesContainer = styled.div`
+    width: 100vw;
+    height: 90vh;
+    position: relative;
+    display: flex;
+    justify-content: center;
+  `
+  const WelcmeContainer = styled.div`
+    position: absolute;
+    z-index: 2;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    background-color: transparent;
+
+    top: 20%;
+
+    & > h1 {
+      text-align: center;
+      margin-bottom: 5vh;
```

Comment:
Try to use % or em / rem to be more flexible when changing the font-size and add 100vh and vw to a global style (if you have one) 😉 . Jonas and Manu requested this in my [Pullrequest](https://github.com/JoscaWij/Spacey/pull/40#discussion_r491929678)